app/views/monthly_statistics/show.html.erb

```diff
@@ -42,11 +42,33 @@
   </div>
 </div>
 
+<div class="govuk-grid-row">
```

Comment:
I wonder if each of these sections should be a separate component?

Reply:
Maybe! Don't feel strongly rn but we could always change later?